### DISCUSSION TITLE
mpv's input.conf

### DISCUSSION THREAD

**Discussion by regs01** [smplayer-dev/smplayer#1083] [Q&A]
Is there any way to make input.conf to work with SMPlayer GUI? I've set `disable_player_config=false` and placed shader files into `/home/user/.var/app/info.smplayer.SMPlayer/config/mpv/shaders` and input.conf into `/home/coth/.var/app/info.smplayer.SMPlayer/config/mpv`. There are several keybinding to enable specific shaders, but it's only working when ticking Run MPlayer/mpv in its own window.

**Comment by qmarsn** [comment]
Was this ever solved? I'm interested in importing my input.conf from MPV into SMPlayer as player.keys so all my shortcuts and shortcuts are mapped back retroactively. Any thoughts would be greatly appreciated.

Thanks!!

**Comment by smplayer-dev** [comment]
The only way is to enable the option "run MPlayer/mpv in its own window."
